include/sst/basic-blocks/dsp/FollowSlewAndSmooth.h

```diff
@@ -142,13 +142,15 @@ struct RunningAverage
     size_t nPoints{0};
     size_t head{0}, tail{0};
     float avg{0}, oneOverN{1};
-    RunningAverage(float *ontoStorage, size_t np) : storage{ontoStorage}, nPoints{np}
+    RunningAverage() {}
+
+    void setStorage(float *s, size_t np)
     {
-        reset();
+        storage = s;
+        nPoints = np;
         oneOverN = 1.0 / (nPoints - 1);
     }
```

Comment:
This shou” call reset like the old ctor did no?

Reply:
And now it does!